network/p2p/p2p_pubsub.go

```diff
@@ -92,6 +93,13 @@ func (n *p2pNetwork) Broadcast(msgID spectypes.MessageID, msg *spectypes.SignedS
 		}
 		committeeID = share.CommitteeID()
 	}
+
+	share := n.nodeStorage.ValidatorStore().Validator(msg.SSVMessage.MsgID.GetDutyExecutorID())
+	if share == nil {
+		return fmt.Errorf("could not find validator: %x", msg.SSVMessage.MsgID.GetDutyExecutorID())
+	}
+	committeeID = share.CommitteeID()
+
```

Comment:
this looks exactly like the above code, if its the same maybe we can remove the condition and the duplication?

Reply:
Removed. Need to clarify this https://github.com/ssvlabs/ssv/pull/1442#discussion_r1674204266

Reply:
@y0sher solved. Removed dupe code.